package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Web feature assets for BCPL",
   "main": "index.js",
   "dependencies": {
+    "angular-aria": "^1.6.9",
```

Comment:
This should be a devdependency since we use the CDN versions of those files (maybe not forever, but at the moment we do).